### PR TITLE
Update screenshot in README

### PR DESCRIPTION
**Pull request by librarychik**
Interface changed slightly, updating screenshot to match.

# Interested parties
 @Islandora/7-x-1-x-committers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Install as usual, see [this](https://drupal.org/documentation/install/modules-th
 
 Configure the path patterns using the usual Pathauto interface, at Administration » Configuration » Search and Metadata » URL Aliases » Patterns (admin/config/search/path/patterns). 
 
-![Pathauto Configuration](https://raw.githubusercontent.com/wiki/Islandora/islandora_pathauto/images/sample-pathauto-configuration.png)
+![pathautoconfiguration](https://user-images.githubusercontent.com/877577/47673120-fbac6700-db70-11e8-98fc-51f5b2cb8555.png)
```
Comment (bryjbrown) [inline README.md]: Awesome job updating the README! One thing to note is that the text between the brackets, despite not displaying, still serves as alt text which screenreaders use, so its a best practice to make sure that this text is readable. For this reason I'd recommend going with the new URL to the new image, but keeping the bracketed text `Pathauto Configuration`.

**Review by bryjbrown — CHANGES_REQUESTED** [review]

**Comment by librarychik** [comment]
@bryjbrown I haven't had a change request on a PR before.  Do I just re-edit the README and start over making the change you suggested? thx ~ j

**Comment by bryjbrown** [comment]
@librarychik Its pretty easy, just make the changes in the same branch in your forked copy of this repository. After committing the changes (and pushing them back up to GitHub if you are doing them locally) the changes will automatically appear in this PR.

**Comment by librarychik** [comment]
@bryjbrown Okay, done (I think).

**Review by bryjbrown — APPROVED** [review]

**Comment by bryjbrown** [comment]
@librarychik Awesome! Looks good.

**Pull request merged**